common/test/db_cache/bok_choy_schema_default.sql

```diff
@@ -456,7 +456,7 @@ CREATE TABLE `auth_user` (
   `is_superuser` tinyint(1) NOT NULL,
   `username` varchar(150) NOT NULL,
   `first_name` varchar(30) NOT NULL,
-  `last_name` varchar(30) NOT NULL,
+  `last_name` varchar(150) NOT NULL,
```

Comment:
@feanil Am I remembering correctly that we decided it was ok for environments other than stage and prod to get the last_name length change?  Not great that our tests don't match prod in that respect.